# Generated by Django 4.2.7 on 2024-01-27 12:44

import cloudinary.models
from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='ReportSwarmCase',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('first_name', models.CharField(max_length=60, null=True)),
                ('last_name', models.CharField(max_length=60, null=True)),
                ('phone', models.CharField(max_length=60, null=True)),
                ('email', models.EmailField(max_length=254, null=True)),
                ('address', models.CharField(max_length=300, null=True)),
                ('town', models.CharField(max_length=60, null=True)),
                ('county', models.CharField(max_length=60, null=True)),
                ('eircode', models.CharField(max_length=8, null=True)),
                ('location', models.CharField(choices=[('outside', 'Outside'), ('inside', 'Inside')], max_length=8)),
                ('description', models.TextField(max_length=10000, null=True)),
                ('image', cloudinary.models.CloudinaryField(max_length=255, verbose_name='image')),
                ('createdOn', models.DateTimeField(auto_now_add=True)),
                ('updatedOn', models.DateTimeField(auto_now=True)),
                ('status', models.CharField(choices=[('Open', 'Open'), ('Closed_Resolved', 'Resolved'), ('Closed_Unresolved', 'Unable to Resolve')], default='Open', max_length=100)),
                ('assignee', models.ForeignKey(default=1, on_delete=django.db.models.deletion.CASCADE, related_name='user', to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
